apps/plotly_examples/scripts/prompts.txt

```diff
@@ -0,0 +1,50 @@
+// Prompt 1 - LLM is given few scenarios on which it builds the storyline for each scenario
```

Comment:
Can you add the python script calling LLM and readme file how to run it.

Reply:
Will add it in the next PR